daemons/dss-checkout-sfn/app.py

```diff
@@ -2,6 +2,7 @@
 import sys
 
 import domovoi
+import logging
```

Comment:
logging is stdlib, btw.

Reply:
Just to clarify... what @ttung means is please move it up into the group of stdlib imports. We group imports into 3 groups (stdlib, third-party, our own) and separate the groups by an extra newline.

Reply:
thanks for the explanation, fixed